src/components/tabController/index.tsx

```diff
@@ -204,4 +237,13 @@ TabController.TabBar = TabBar;
 TabController.TabBarItem = TabBarItem;
 TabController.TabPage = TabPage;
 TabController.PageCarousel = PageCarousel;
-export default asBaseComponent(TabController);
+export default asBaseComponent<
+  TabControllerProps,
+  {
+    TabBar: typeof TabBar;
+    TabBarItem: typeof TabBarItem;
+    TabPage: typeof TabPage;
+    PageCarousel: typeof PageCarousel;
+  }
```

Comment:
What about doing `typeof TabController` like we usually do. 
Wouldn't that work?

Reply:
Yes, my mistake, fixed